scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -223,7 +225,7 @@ task ExtractFromSuperpartitionedTables {
     }
 
     runtime {
-        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:409.0.0-alpine"
+        docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-03-07-alpine-v2"
```

Comment:
this one is v2 and the others are not, is this intentional?

Reply:
thanks — made them consistent